pick_kernel/kernel.py

```diff
@@ -229,6 +236,44 @@ def display(self, obj, parent, display_id=False, update=False):
             ident=self._topic("display_data"),
         )
 
+    def _publish_error(self, exc_info, parent=None):
```

Comment:
Do these need to be marked as private? Seems like a method that registered config systems may want to use.

Reply:
This is for our "parent" kernel, so I don't want to expose a public interface for it.